<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vaccine_appointment</class>
 <widget class="QWidget" name="vaccine_appointment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1654</width>
    <height>718</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>疫苗接种预约</string>
  </property>
  <widget class="QTableWidget" name="vaccine_tb">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>1081</width>
     <height>201</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>街道号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>街道</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>区</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>疫苗公司</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第几针</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>医院名字</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>疫苗接种预约</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前，你已接种</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="personal_tb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>company</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提供接种服务的医院及疫苗类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请在右表中选择服务</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择日期</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>861</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hour">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>430</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>08</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>09</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>290</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>时间：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="min">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>430</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>420</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>580</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定预约</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>580</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>19</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>点击初始化</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
